### PR TITLE
Make text clickable for radio buttons in diagnosis code page

### PR DESCRIPTION
**Pull request by ibeckermayer**
Gives each radio button an id and makes use of the `<label>` tag's `for` attribute to cause clicking on the label to select the radio button (https://stackoverflow.com/a/1146936)

closes https://github.com/covidwatchorg/portal/issues/464

### CODE REVIEW

File: frontend/client/styles/screens/code_validations.scss [hunk 1]
```diff
@@ -91,6 +94,10 @@
 
         .col-2-sub-header {
           color: $darkMediumGray;
+          font-family: Montserrat;
```
Comment (colbymorrison) [inline frontend/client/styles/screens/code_validations.scss:97]: Can remove this line. Montserrat is set by default in `application.scss`

File: frontend/client/styles/screens/code_validations.scss [hunk 2]
```diff
@@ -91,6 +94,10 @@
...
+          font-size: 14px;
+          font-weight: normal;
```
Comment (colbymorrison) [inline frontend/client/styles/screens/code_validations.scss:99]: Remove this line and line below (99-100), these are the defaults

**Comment by ibeckermayer** [comment]
> See comments. Also, it is not due to this PR but the white circle in the radio button looks off-centered in a way it didn't before. Did this get changed in another PR?
> 
> radio

Browser/OS?

**Comment by ibeckermayer** [comment]
> > See comments. Also, it is not due to this PR but the white circle in the radio button looks off-centered in a way it didn't before. Did this get changed in another PR?
> > radio
> 
> Browser/OS?

https://github.com/covidwatchorg/portal/issues/477

**Pull request merged**